SETUP.md

```diff
@@ -42,6 +42,8 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
   * This may fail if your are on a Mac and your OSX XCode Command Line Tools were not installed properly. See Bundle Install Tips for more information.
 1. (Optional, Code.org engineers only) Setup AWS - Ask a Code.org engineer how to complete this step
    1. Some functionality will not work on your local site without this, for example, some project-backed level types such as https://studio.code.org/projects/gamelab. This setup is only available to Code.org engineers for now, but it is recommended for Code.org engineers.
+1. Run the website `bin/dashboard-server`
```

Comment:
Nice add! I can't believe we didn't include this before. Is it worth adding something about also starting yarn?

Reply:
As we talked about offline, I agree that since `yarn start` is mentioned in the `apps/` sub directory, that is enough. That's more an "apps" development improvement someone would be interested in if they were making changes in that directory.